test/cholesky/lkjcholesky.jl

```diff
@@ -124,14 +124,21 @@ using FiniteDifferences
     end
 
     @testset "properties" begin
-        @testset for p in (4, 5), η in (2, 3.5), uplo in ('L', 'U')
+        @testset for p in (4, 5), η in (0.5, 2, 3.5), uplo in ('L', 'U')
```

Comment:
Maybe include `1` to test at the boundary.
```suggestion
        @testset for p in (4, 5), η in (0.5, 1, 2, 3.5), uplo in ('L', 'U')
```